tests/core/fixtures.py

```diff
@@ -139,6 +146,16 @@ def chain(chaindb, funded_address, funded_address_initial_balance):  # noqa: F81
         "deployed_address": PAYGAS_contracts['PAYGAS_contract_triggered_twice']['address'],
         "initial_balance": 100000000,
     },
+    {
+        "contract_code": nonce_tracking_contracts['nonce_tracking_contract']['bytecode'],
+        "deployed_address": nonce_tracking_contracts['nonce_tracking_contract']['address'],
+        "initial_balance": 100000000,
+    },
+    {
+        "contract_code": nonce_tracking_contracts['no_nonce_tracking_contract']['bytecode'],
+        "deployed_address": nonce_tracking_contracts['no_nonce_tracking_contract']['address'],
+        "initial_balance": 100000000,
```

Comment:
You can replace all `100000000` with `funded_address_initial_balance` fixture now.